src/lib/reasoners/satml.ml

```diff
@@ -1857,4 +1859,7 @@ module Make (Th : Theory.S) : SAT_ML with type th = Th.t = struct
       end;
     enqueue env g.neg 0 None
 
+  let optimize env ~to_max obj =
+    (* TODO: Should we add the objective function to unit_tenv? *)
```

Comment:
No, the objective function is only relevant when the environment is used for splitting, and the unit environment is not used for splitting.